installer/tue-install-impl.bash

```diff
@@ -101,7 +101,12 @@ function tue-install-target
     tue-install-debug "tue-install-target $*"
 
     local target=$1
-    local now=$2
+
+    if [[ "$ROS_PYTHON_VERSION" == 3 && "$target" == python-* ]]
+    then
+        tue-install-error "Trying to install a python2 target (${target}) while the ROS default is python3. Fix your dependencies"
+        return 3
+    fi
```

Comment:
How about `python-*-pip` targets. Not all will be migrated to `python3-*-pip` in rosdep, which we follow.

Reply:
Simply open a PR there, they get accepted easily :slightly_smiling_face: 

But sure, you can remove the check if it doesn't suit the tooling.

Reply:
I did, https://github.com/ros/rosdistro/pull/28264. So they are not accepting python3 versions of existing `python-*-pip` targets. And are not accepting new `python-*-pip` targets either, only `python3-*-pip` (https://github.com/ros/rosdistro/pull/28672). So you get a big mess of `python-*-pip` and `python3-*-pip`.

Reply:
Hah, this way the deprecation will never go away because people can't use the new format.

Alrighty then... This should be gone.